common/tables/table-en.html

```diff
@@ -0,0 +1,622 @@
+<table class="table tbl-gridify">
+	<thead>
+		<tr>
+			<th>Number</th>
+			<th>Column heading</th>
+			<th>Column heading</th>
+			<th>Column heading</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr class="active">
+			<th>1</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr>
+			<th>2</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr class="success">
+			<th>3</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr>
+			<th>4</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr class="info">
+			<th>5</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr>
+			<th>6</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr class="warning">
+			<th>7</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr>
+			<th>8</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr class="danger">
+			<th>9</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+	</tbody>
+</table>
+<pre><code>&lt;table class="table tbl-gridify"&gt;
+...
+&lt;/table&gt;</code></pre>
+
+<h2> Tables within panels </h2>
+<p>Full-width content:</p>
+	<div class="panel panel-default">
+		<table class="table">
+		<caption class="wb-inv">Caption text</caption>
+		<thead>
+			<tr>
+				<th>Table header</th>
+				<th>Table header</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td>Table data</td>
+				<td>Table data</td>
+			</tr>
+			<tr>
+				<td>Table data</td>
+				<td>Table data</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+<p>Padded plus full-width content:</p>
+<div class="panel panel-default">
+	<div class="panel-body">
+		<p>Panel body Content</p>
+	</div>
+	<table class="table">
+		<caption class="wb-inv">
+			Caption  text
+		</caption>
+		<thead>
+			<tr>
+				<th>Table header</th>
+				<th>Table header</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td>Table data</td>
+				<td>Table data</td>
+			</tr>
+			<tr>
+				<td>Table data</td>
+				<td>Table data</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+<pre><code> //Full-width content, invisible caption:
+&lt;div <strong>class=&quot;panel panel-default&quot;</strong>&gt;
+<strong>&lt;table class=&quot;table&quot;&gt;</strong>
+<strong>&lt;caption class=&quot;wb-inv&quot;&gt;</strong>Caption text&lt;/caption&gt;
+	...
+&lt;/table&gt;
+&lt;/div&gt;
+
+//Padded plus full-width content:
+<strong>&lt;div class=&quot;panel panel-default&quot;&gt;
+&lt;div class=&quot;panel-body&quot;&gt;</strong>
+<strong>&lt;p&gt;</strong>...&lt;/p&gt;
+&lt;/div&gt;
+<strong>&lt;table class=&quot;table&quot;&gt;</strong>
+	...
+&lt;/table&gt;
+&lt;/div&gt;</code></pre>
```

Comment:
For consistency with the other code sample on this page, simply remove the emphasis. 

But this example are not using the panel according to its design specification, see my other comment on that subject

Reply:
This was addressed